apps/test-suite/tests/Camera.js

```diff
@@ -95,8 +95,14 @@ export async function test(t, { setPortalChild, cleanupPortal }) {
       });
 
       t.it(
-        'returns Base64 only if requested, and base64 result does not contain newline character',
+        'this long description is forcing us to move testing method signature to the next line, but let's keep method body distinguishable by keeping indentation anyway',
         async () => {
+          await mountAndWaitFor(<Camera ref={refSetter} style={style} />);
+          let picture = await instance.takePictureAsync({ base64: false });	
+          t.expect(picture).toBeDefined();
+          ...
+        }
+      )
```

Comment:
Well, I'm afraid that is not what I was suggesting 😅
Sorry for using `suggestion` block for this one ... 😞
I just wanted to present my idea 😉
Can you please fix this one?

Reply:
ah 😅 got it